src/components/Button/src/Button.vue

```diff
@@ -227,13 +266,12 @@ export default {
 	cursor: pointer;
 	transition:
 		color 0.2s ease-in,
-		background-color 0.2s ease-in;
+		background-color 0.2s ease-in,
```

Comment:
Curious, do we share any variables between components in Maker? for example, would we add a `hover-transition` in a shared spot and use between component?

Reply:
nope, we don't have any CSS sharing yet, that's a TODO that's available for anyone to tackle